src/braket/circuits/gate.py

```diff
@@ -49,18 +73,38 @@ def to_ir(self, target: QubitSet) -> Any:
         Returns:
             IR object of the quantum operator and target
         """
+        if self._adjoint:
+            # Use adjoint list of original gate
+            # After all, we want the adjoint of the original, not the adjoint of the adjoint!
+            return [elem.to_ir(target) for elem in self.adjoint().adjoint_list()]
+        return self._to_ir(target)
+
+    def _to_ir(self, target: QubitSet) -> Any:
         raise NotImplementedError("to_ir has not been implemented yet.")
 
+    @property
+    def ascii_symbols(self) -> Tuple[str, ...]:
+        """Tuple[str, ...]: Returns the ascii symbols for the quantum operator."""
+        if self._adjoint:
+            return tuple(
+                # "C" stands for control, and doesn't need a dagger symbol
```

Comment:
But the ascii symbols are determined by subclasses, it seems like this is information leaking to know what one of the subclass ascii symbols would be.

Reply:
Why can't the subclass just override this method?

Reply:
I want to avoid having to copy the `if` statement for every subclass, as it's easy for a developer to forget to add the branch,